client/js/libs/handlebars/friendlydate.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+const moment = require("moment");
+
+module.exports = function(time) {
+	// See http://momentjs.com/docs/#/displaying/calendar-time/
+	return moment(new Date(time)).calendar(null, {
+		sameDay: "[Today]",
+		lastDay: "[Yesterday]",
+		lastWeek: "L", // Locale
```

Comment:
This is just a general comment, not a comment on you.

I hate where we have to comment on these stupid magic shortcodes to figure out what on earth we are wanting to show. It's ridiculous. Why can't they have a better way of writing these out so it's self-documenting? Gagh!